utils/gear-runtime-test-cli/src/command.rs

```diff
@@ -238,6 +238,8 @@ macro_rules! command {
                 .map(|(k, v)| (H256::from_slice(k.as_ref()), *v))
                 .collect();
 
+            let block_number = System::block_number() as u32;
```

Comment:
for these tests we can also use some defaults

Reply:
instead of system query

Reply:
done